packages/web-app-files/src/components/SideBar/SideBar.vue

```diff
@@ -181,7 +183,7 @@ export default {
       return null
     },
     isSingleResource() {
-      return !this.areMultipleSelected && !this.isRootFolder
+      return !this.areMultipleSelected && (!this.isRootFolder || this.highlightedFileIsSpace)
```

Comment:
I guess you wanted to have the following:
```suggestion
      return !this.areMultipleSelected && (!this.isRootFolder || !this.highlightedFileIsSpace)
```

Reply:
What do you think about modifying `isRootFolder` to also return `true` when the highlighted file is a space? That might solve more things automagically. Semantically, the space root is the same as the personal storage root and a public link root. If you want to fix a bug on the way, return `true` for `isRootFolder` for public link roots as well. ;-)

Reply:
That would change the logic here though. A space is supposed to be a single Resource, the rootFolder would be the "entry point" for all spaces IMO. Otherwise I'd have to adjust several other condition checks. (like the class check e.g. `'resource-info-hidden': !isSingleResource`)

Reply:
Then keep it like it is. :-) Then your initial implementation is already correct 👍